uqbar.wit

```diff
@@ -142,11 +142,11 @@ interface standard {
 
     // check to see if the sender of a prompting message has a given capability, issued by us
     // if the prompting message has a remote source, they must have attached it.
-    has-capability: func(params: json) -> bool;
+    has-capability: func(process: process-id, params: json) -> bool;
```

Comment:
there's an argument to be made that maybe this should be `address: address` instead of process-id. Should you be able to check if someone sent you a cap from another node? That might be useful one day?